packages/@react-aria/spinbutton/src/useSpinButton.ts

```diff
@@ -128,15 +128,7 @@ export function useSpinButton(
   // This ensures that macOS VoiceOver announces it as "minus" even with other characters between the minus sign
   // and the number (e.g. currency symbol). Otherwise it announces nothing because it assumes the character is a hyphen.
   // In addition, replace the empty string with the word "Empty" so that iOS VoiceOver does not read "50%" for an empty field.
-  textValue =
-    textValue === '' ?
-    stringFormatter.format('Empty') :
-    (textValue || `${value}`)
-    .trim()
-    .replace('-', '\u2212')
-    // Replace negative value formatted using currencySign: 'accounting'
-    // with a value that can be announced using a minus sign.
-    .replace(/^\((.+)\)$/, '\u2212$1');
+  textValue = textValue === '' ? stringFormatter.format('Empty') : (textValue || `${value}`).replace('-', '\u2212');
```

Comment:
This should also be moved up to useNumberField, because then we can grab the minusSign character and out of `formatToParts`

Reply:
I think the problem was that it _is_ coming from the number formatter, but the screen reader doesn't announce that character right.

Reply:
right, but there may be other minus sign symbols, if we moved it up, we could always replace whatever symbol with the one that we know the screen reader can announce?

Reply:
Shouldn't `useSpinButton` handle announcement of the minus sign and "Empty" at this lower level? I can understand moving the currency formatting up, the useNumberField, because that's where we accept `formatOptions`, but it seems to make sense to keep the logic to announce 'minus' here. Are you saying there are three Unicode symbols for the minus sign that screen readers can announce, or three locales that use different symbols for the minus sign, that might have to be replaced with a more robust regex?

Reply:
There are around 3 different locale specific minus signs, if I recall correctly.
But now I feel like I'm expanding the scope of this.
So let's keep useSpinButton as it was, and handle the currency negative up in useNumberField as you were doing.